README.md

```diff
@@ -1 +1,166 @@
-# Hello_Microverse_Project
\ No newline at end of file
+<a name="Hello Microverse Project"></a>
+
+<div align="center">
+  
+
+  <h3><b>Hello Microverse project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> This is my portfolio I build it with html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+1. html
+2. css
+3. Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+Added index.html file.
+Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-demo will be added soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+https://github.com/albertkantwi/Hello_Microverse_Project
+
```

Comment:
> - [ ] Please include the following command as per the code reviewer's request: `git clone https://github.com/albertkantwi/Hello_Microverse_Project.git`. It is important to provide the actual command instead of just the link when cloning the repository. This helps ensure clarity and accuracy in the instructions.